include/mg_procedure.h

```diff
@@ -164,6 +164,12 @@ enum mgp_value_type {
   MGP_VALUE_TYPE_DURATION,
 };
 
+/// All available log levels that can be use in mgp_log function
+MGP_ENUM_CLASS mgp_log_level{
+    MGP_LOG_LEVEL_TRACE, MGP_LOG_LEVEL_DEBUG, MGP_LOG_LEVEL_INFO,
+    MGP_LOG_LEVEL_WARN,  MGP_LOG_LEVEL_ERROR, MGP_LOG_LEVEL_CRITICAL,
+};
+
```

Comment:
I this file up to my knowledge the functions are organized into different groups. The smallest group is `Error Codes`, check out the start and end comments of the group. It is important because the docs page are generated based on these comments.
```

/// @name Error Codes
///
///@{

/// All functions return an error code that can be used to figure out whether the API call was successful or not. In
/// case of failure, the specific error code can be used to identify the reason of the failure.
MGP_ENUM_CLASS MGP_NODISCARD mgp_error{
    MGP_ERROR_NO_ERROR,           MGP_ERROR_UNKNOWN_ERROR,
    MGP_ERROR_UNABLE_TO_ALLOCATE, MGP_ERROR_INSUFFICIENT_BUFFER,
    MGP_ERROR_OUT_OF_RANGE,       MGP_ERROR_LOGIC_ERROR,
    MGP_ERROR_DELETED_OBJECT,     MGP_ERROR_INVALID_ARGUMENT,
    MGP_ERROR_KEY_ALREADY_EXISTS, MGP_ERROR_IMMUTABLE_OBJECT,
    MGP_ERROR_VALUE_CONVERSION,   MGP_ERROR_SERIALIZATION_ERROR,
};
///@}
```

Currently this `mgp_log_level` enum is in the `Operations on mgp_value` group, which is not good in my opinion. I would this into the `Query Module & Procedures`, together with the function.

Reply:
Yes, I completely agree 👍🏻 I didn't know actually that the docs page is generated based on these comments. Fixed 👍🏻

Reply:
Sorry for not being complete, I also missed that previously, but the `mgp_log` function is in the `Memgraph Magic Functions API` group, so that should be moved also.